### DISCUSSION TITLE
Support for sending "resource" parameter during authentication for ADFS OIDC integration

### DISCUSSION THREAD

**Discussion by RichardLourens** [outline/outline#6362] [Ideas]
Hi,

I've been encountering some issues while trying to connect outline with our on premise ADFS instance. Microsoft requires a custom "resource" parameter to be send for authentication as can be seen here (https://learn.microsoft.com/en-us/windows-server/identity/ad-fs/development/ad-fs-openid-connect-oauth-concepts) or else it will fail with the following errors:

`"Failed to parse response from https://adfs.test.local/adfs/userinfo. Expected JSON, got: "`
and
`Bearer error="invalid_token", error_description="MSIS9921: Received invalid UserInfo request. Audience 'microsoft:identityserver:[client_identifier]' in the access token is not same as the identifier of the UserInfo relying party trust 'urn:microsoft:userinfo'."`

I believe this is the same issue as [#4491](https://github.com/outline/outline/discussions/4491), but as far as i know, there hasn't been any solution to this yet. I'd like to know if there is any support planned to add a custom resource path in the .env OIDC-config to support OIDC authentication for ADFS.

**Comment by tommoor** [comment]
There aren't really any plans around this as there are already two other options when it comes to Microsoft login – both OAuth and SAML work with AD.
